boards/vcu108/vcu108.core

```diff
@@ -2,6 +2,8 @@ CAPI=1
 [main]
 name = wallento:boards:vcu108
 depend =
+  wallento:boards:vcu108fancontrol
+  wallento:boards:vcu108system_management_wiz
   wallento:boards:vcu108-ddr
```

Comment:
Please name the two new modules something like vcu108-fancontrol to make it a bit more similar with what we have currently.